app/controllers/cards_controller.rb

```diff
@@ -38,6 +38,18 @@ def destroy
     @card.destroy
     redirect_to cards_path
   end
+  
+  def check
+    @card = Card.find(params[:card][:id])
+    if  params[:card][:translated_text] == params[:answer]
+      flash[:success] = "Правильно"
+      add_days
+      redirect_to root_url 
+    else
+      flash.now[:danger] = "Не правильно, попробуй еще раз"
+      render 'home/index'     
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Trailing whitespace detected.